tests/sweep_framework/sweep_utils/sharding_utils.py

```diff
@@ -68,6 +72,11 @@ def gen_sharded_spec_unary(num_shapes, max_tensor_size=4 * 1024 * 1024, layouts=
                 physical_shape[1] *= min_shard_size_y
                 physical_shape[0] *= min_shard_size_x
 
+                if shard_orientation == "ROW_MAJOR":
+                    tmp = physical_shape[-2]
```

Comment:
More pythonic way would be: 
physical_shape[-1], physical_shape[-2] = physical_shape[-2], physical_shape[-1]

feel free to ignore